front50-sql/src/main/kotlin/com/netflix/spinnaker/config/SqlConfiguration.kt

```diff
@@ -58,11 +56,11 @@ class SqlConfiguration {
     )
 
   @Bean
-  @ConditionalOnBean(name = ["secondaryJooq"])
+  @ConditionalOnProperty("sql.enabled", "sql.secondary.enabled")
   fun secondarySqlStorageService(
     objectMapper: ObjectMapper,
     registry: Registry,
-    @Qualifier("secondaryJooq") jooq: DSLContext,
+    jooq: DSLContext,
```

Comment:
rmm, wont' this use the primary DSLContext without the qualifier?  NOT dug yet just... this seems to be an interesting change...

Reply:
secondaryJooq is conditionally created when the dialect doesnt match the default pool. I have pushed a change that when secondaryJooq exists it is used as the effective DSLContext. When it doesnt exist then it defaults to the jooq DSLContext.